test/corpus/expressions.txt

```diff
@@ -405,35 +366,29 @@ var x = new A
         type: (implicit_type)
         (variable_declarator
           name: (identifier)
-          (equals_value_clause
-            (object_creation_expression
-              type: (identifier)
-              initializer: (initializer_expression
-                (assignment_expression
-                  left: (identifier)
-                  (assignment_operator)
-                  right: (integer_literal))
-                (assignment_expression
-                  left: (element_binding_expression
-                    (bracketed_argument_list
-                      (argument
-                        (identifier))))
-                  (assignment_operator)
-                  right: (integer_literal)))))))))
+          (object_creation_expression
+            type: (identifier)
+            initializer: (initializer_expression
+              (assignment_expression
+                left: (identifier)
+                right: (integer_literal))
+              (assignment_expression
+                left: (element_binding_expression
+                  (argument
+                    (identifier)))
+                right: (integer_literal))))))))
   (global_statement
     (expression_statement
       (assignment_expression
         left: (parenthesized_expression
           (identifier))
-        (assignment_operator)
         right: (integer_literal))))
   (global_statement
     (expression_statement
-      (assignment_expression
-        left: (prefix_unary_expression
-          (identifier))
-        (assignment_operator)
-        right: (integer_literal)))))
+      (prefix_unary_expression
```

Comment:
Subtle change here with the assignment now being a prefix_unary_expression rather than the left term.

Reply:
good catch - that original splitting of _pointer_indirection_expression was very costly due to the conflict - but if we only allow lvalue_expression's there then it's much better